src/main/resources/com/puppycrawl/tools/checkstyle/messages.properties

```diff
@@ -19,3 +19,4 @@ PackageObjectFactory.unableToInstantiateExceptionMessage=Unable to instantiate '
    Please recheck that class name is specified as canonical name or read how to configure \
    short name usage https://checkstyle.org/config.html\#Packages. Please also \
    recheck that provided ClassLoader to Checker is configured correctly.
+parse.exception=TreeWalker is skipped due to a parse exception - {0}
```

Comment:
User set skipFileOnJavaParseException but we tell him about Treewalker.
Something is mismatching.

Reply:
We need something like:
Java specific module are skipped due to parse exception 

In this case user clearly match what he activated with new behavior

Reply:
> Java specific module are skipped due to parse exception

what do you mean by "Java specific module" ? do you want this exact message? can we be more explicit and say:
Treewalker Checks is skipped due to parse exception?

Reply:
done